service-commands/src/commands/seed/SeedSession.js

```diff
@@ -1,26 +1,18 @@
-const AudiusLibs = require('@audius/libs')
```

Comment:
can ignore anything in service-commands. only cherry picked some commits for local testing to work